code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -1977,4 +1977,54 @@
 	name = "Nanoblood"
 	result = /datum/reagent/nanoblood
 	required_reagents = list(/datum/reagent/dexalinp = 1, /datum/reagent/iron = 1, /datum/reagent/blood = 1)
-	result_amount = 3
\ No newline at end of file
+	result_amount = 3
+
+// New recipes as of 15th October, 2020. - Lion
+
+/datum/chemical_reaction/phlogiston
```

Comment:
where did you get these from, and what do they do?

Reply:
As per the Changelog:
  - rscadd: "Chemical Reagent: Phlogiston, which upon mixing, unleashes a spreading and debilitating Phoron fire. Made with 1u Aluminum, Phoron and Sulphuric Acid, results in 1u."
  - rscadd: "Chemical Reagent: Vecuronium Bromide, one of the most powerful paralytics. It will reduce someone's voice to whispers and will near-instantly paralyze them. Made with 1u Ethanol, and 2u of Mercury and Hydrazine. Results in 2u."


These are from the newer Bay fork that I have, and were to compensate for the axe that the acids got, as it hamstrung medical/sci-based unitologists in what weaponry was readily available. Besides, this adds diversity.